app/models/pipeline_run_stage.rb

```diff
@@ -139,9 +138,7 @@ def host_filtering_command
     if sample.s3_bowtie2_index_path.present?
       batch_command_env_variables += " BOWTIE2_GENOME=#{sample.s3_bowtie2_index_path} "
     end
-    batch_command = "aws s3 cp #{IdSeqPipeline::S3_HOST_FILTER_SCRIPT_LOC} .; chmod 755 #{script_name}; " \
-      "aws s3 cp #{IdSeqPipeline::S3_COMMON_SCRIPT_LOC} .; chmod 755 #{common_script_name}; " +
-                    batch_command_env_variables + " ./#{script_name}"
+    batch_command = batch_command_env_variables + " idseq_pipeline host_filtering"
```

Comment:
nice, but maybe for the next diff we should move from environment variables to command arguments.